docs/types/operators.rst

```diff
@@ -41,7 +44,8 @@ and 256 exceeds the range allowed for this type.
 
 Another consequence is that an expression like ``1.5 + 1.5`` is valid but ``1.5 + (true ? 1.5 : 2.5)`` is not.
 This is because the former is a rational expression evaluated in unlimited precision and only its final value matters.
-The latter involves a conversion of a fractional rational number to an integer, which is currently disallowed.
+The latter involves a conversion of a fractional rational number to an unsigned fixed point type (``ufixed8x1``, the
```

Comment:
The main problem here is that the implementation of fixed point types in Solidity is not finished. We decided to take them out of the language again, so maybe it is better to not talk about this in detail and just remove fixed point types from the language altogether.